enterprise/app/org/org_form.tsx

```diff
@@ -190,6 +190,23 @@ export default abstract class OrgForm<T extends GroupRequest> extends React.Comp
             <span>Default to self-hosted executors</span>
           </label>
         )}
+        {/* <label className="form-row input-label">
```

Comment:
Instead of commenting, what do you think about putting this all behind a flag so it can be enabled / disabled more easily?

Reply:
Done! `--app.user_owned_keys_enabled`